specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -149,6 +149,11 @@
             "$ref": ".//examples//SuccessfulAnalyzeStatusRequest.json"
           }
         },
+        "x-ms-pageable": {
+          "operationName": "AnalyzeStatusNextPage",
+          "nextLinkName": "nextLink",
```

Comment:
For some of these next links, the name should actually be `@nextLink` based off of swagger definitions

Reply:
@maririos I think what @iscai-msft is trying to say is "nextLink" should be a property name defined [here](https://github.com/Azure/azure-rest-api-specs/blob/cbaf1c13997c8159cc10615fea27b32c20db676c/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L2328), which in this case is `"@nextLink"`, so we should update `nextLink` to `@nextLink`.

Reply:
@maririos I think what @iscai-msft is trying to say is "nextLink" should be a property name defined in response object [here](https://github.com/Azure/azure-rest-api-specs/blob/cbaf1c13997c8159cc10615fea27b32c20db676c/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L2328), which in this case is `"@nextLink"`, so we should update `nextLink` to `@nextLink`.